### DISCUSSION TITLE
Jenkins and Rancher Desktop

### DISCUSSION THREAD

**Discussion by espinoleandroo** [rancher-sandbox/rancher-desktop#3481] [Q&A] [answered]
Hi, I'm trying to create a docker image in a Jenkins pipeline, but _Jenkins doesn't recognize the **docker** command_.
My work environment is:
- Windows 10
- Rancher Desktop 0.7.1
- Jenkins  2.361.3 (in windows)

Somebody did have this problem? Or figure out a solution for it?

**Comment by adamkpickering** [comment]
Does it work if you upgrade to the latest version of Rancher Desktop? RD has changed so much since 0.7.1 - it would be pointless to try to debug it.

**Comment by espinoleandroo** [comment, marked as answer]
Hi I've discovered that my problem was that Jenkins run the docker command in a distinct environment, so here el docker command is no recognized, it happens because I used the Jenkins Windows Installer, with the Jenkins WAR everything is okay using the Rancher Desktop 0.7.1 and newest.

I hope my comment help someone
